xcodeproj/internal/xcodeproj_incremental_rule.bzl

```diff
@@ -635,6 +635,16 @@ Are you using an `alias`? `xcodeproj.focused_targets` and \
         ctx.attr.default_xcode_configuration or xcode_configurations[0]
     )
 
+    # TODO: Remove once we drop support for Xcode 16.x.
```

Comment:
We can generate with Xcode 16.1 but run on Xcode 16.3. I think we need to have both binaries always available, and change which one we use in `bazel_build.sh` based on the Xcode version environment variables.

Reply:
Updated to pass a `legacy_index_import` attr to both the `legacy` and `incremental` generators. Then in `ImportIndexstores.swift` we pick the right one based on `XCODE_VERSION_ACTUAL`